graylog2-web-interface/src/views/components/searchbar/queryinput/StyledAceEditor.jsx

```diff
@@ -8,140 +8,140 @@ const StyledAceEditor = styled(AceEditor).attrs(({ aceTheme, theme }) => ({
+    }
+
+    &.ace_multiselect .ace_selection.ace_start {
+      box-shadow: 0 0 3px 0 ${scTheme.color.global.inputBackground};
+    }
+
+    .ace_marker-layer .ace_step {
+      background: ${scTheme.color.variant.warning};
+    }
+
+    .ace_marker-layer .ace_bracket {
+      margin: -1px 0 0 -1px;
+      border: none;
+    }
+
+    .ace_marker-layer .ace_active-line {
+      background: ${scTheme.color.global.inputBackground};
+    }
+
+    .ace_gutter-active-line {
+      background-color: ${scTheme.color.global.inputBackground};
+    }
+
+    .ace_marker-layer .ace_selected-word {
+      border: 1px solid ${scTheme.color.gray[80]};
+    }
+
+    .ace_invisible {
+      color: ${scTheme.color.global.inputBackground};
+    }
+
+    .ace_keyword,
+    .ace_meta,
+    .ace_storage,
+    .ace_storage.ace_type,
+    .ace_support.ace_type {
+      color: ${scTheme.color.variant.primary};
+    }
+
+    .ace_keyword.ace_operator {
+      color: ${scTheme.color.variant.darker.info};
+    }
+
+    .ace_constant.ace_character,
+    .ace_constant.ace_language,
+    .ace_constant.ace_numeric,
+    .ace_keyword.ace_other.ace_unit,
+    .ace_support.ace_constant,
+    .ace_variable.ace_parameter {
+      color: ${scTheme.color.variant.dark.danger};
+    }
+
+    .ace_constant.ace_other {
+      color: ${scTheme.color.variant.default};
+    }
+
+    .ace_invalid {
+      color: ${util.readableColor(scTheme.color.brand.primary)};
+      background-color: ${scTheme.color.brand.primary};
+    }
+
+    .ace_invalid.ace_deprecated {
+      color: ${util.readableColor(scTheme.color.brand.primary)};
+      background-color: ${scTheme.color.variant.dark.primary};
+    }
+
+    .ace_fold {
+      background-color: ${scTheme.color.variant.info};
+      border-color: ${util.contrastingColor(scTheme.color.global.inputBackground, 'AAA')};
+    }
+
+    .ace_entity.ace_name.ace_function,
+    .ace_support.ace_function,
+    .ace_variable,
+    .ace_term {
+      color: ${scTheme.color.variant.info};
+    }
+
+    .ace_support.ace_class,
+    .ace_support.ace_type {
+      color: ${scTheme.color.variant.dark.warning};
+    }
+
+    .ace_heading,
+    .ace_markup.ace_heading,
+    .ace_string {
+      color: ${scTheme.color.variant.dark.success};
+    }
+
+    .ace_entity.ace_name.ace_tag,
+    .ace_entity.ace_other.ace_attribute-name,
+    .ace_meta.ace_tag,
+    .ace_string.ace_regexp,
+    .ace_variable {
+      color: ${scTheme.color.brand.primary};
+    }
+
+    .ace_comment {
+      color: ${scTheme.color.gray[60]};
+    }
+
+    .ace_indent-guide {
+      background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZgbYnAAAAE0lEQVQImWP4////f4bdu3f/BwAlfgctduB85QAAAABJRU5ErkJggg==) right repeat-y;
+    }
+
+    .ace_content,
+    .ace_placeholder {
+      top: 6px;
+      font-size: 16px;
+      padding: 0 !important;
+      font-family: inherit !important;
+    }
   }
 
   .ace_editor.ace_autocomplete {
```

Comment:
@kyleknighted I like your changes regarding the highlight! While testing these changes, I noticed two problems with the autocomplete:

Due to the new changes the autocomplete lost its top "margin":
![image](https://user-images.githubusercontent.com/46300478/81658750-58146600-9439-11ea-9eae-d573d8ee9126.png)
Compared with before (3.3 beta1):
![image](https://user-images.githubusercontent.com/46300478/81658840-6e222680-9439-11ea-8989-654444e69fc6.png)

And as you can see in the screenshot the autocomplete has a larger width. While having a look at the described problem, I saw the `  .ace_editor.ace_autocomplete {` styling is no longer being included. This is not directly related to this PR, but could you have a look at it as well?

Reply:
got it fixed up

![image](https://user-images.githubusercontent.com/122591/81726384-e80fdb00-944c-11ea-8da2-274e9381ee33.png)